ext/standaloneusers/sql/auto_install.sql

```diff
@@ -46,26 +46,33 @@ ENGINE=InnoDB;
 
 -- /*******************************************************
 -- *
--- * civicrm_user
+-- * civicrm_uf_match
 -- *
--- * A standalone user account
+-- * Standalone User Account. In Standalone this is a superset of the original civicrm_uf_match table.
 -- *
 -- *******************************************************/
-CREATE TABLE `civicrm_user` (
+CREATE TABLE `civicrm_uf_match` (
```

Comment:
We have situations where the extension is not enabled automatically after the Civi upgrade. So when we install the extension via the UI after the upgrade the statement will fail with table already exists error?

Reply:
No it won't. Look a few lines above in this file.

Reply:
So it's going to drop and create, will that cause any issue for already logged-in user who will be running upgrader from the UI.

Reply:
I think there's some confusion here. To clarify, this file is an `install.sql` file - it only runs once, during initial installation. It never runs again, not even during upgrades.